pipelines/environments/mldesigner-minimal-2.0-ubuntu20.04-py38-cpu copy/context/conda_dependencies.yaml

```diff
@@ -0,0 +1,9 @@
+name: mldesigner-minimal
+channels:
+- anaconda
+- conda-forge
+dependencies:
+- python=3.8.13
+- pip=21.3.1
+- pip:
+  - mldesigner=={{latest-pypi-version:pre}}
```

Comment:
If the only difference between this environment and the non-minimal version is azure-ai-ml and azureml-mlflow, does this environment need to exist?

Reply:
Yes, `mldesigner` can work without `azure-ai-ml` and `azureml-mlflow`. and we recommend users to only use `mldesigner` if they do not require any advanced features in a command component.